lib/utils.pm

```diff
@@ -421,6 +422,27 @@ sub is_bridged_networking {
     return $ret;
 }
 
+=head2 set_hostname
+    set_hostname($hostname);
```

Comment:
please insert one blank line in before. It's a more common style, e.g. see the previous function workaround_type_encrypted_passphrase()